src/FSharp.Data.GraphQL.Server.AspNetCore/GraphQLOptions.fs

```diff
@@ -0,0 +1,22 @@
+namespace FSharp.Data.GraphQL.Server.AspNetCore
+
+open FSharp.Data.GraphQL
+open System
+open System.Text.Json
+open System.Threading.Tasks
+open Microsoft.AspNetCore.Http
+
+type PingHandler =
+  IServiceProvider -> JsonDocument option -> Task<JsonDocument option>
```

Comment:
Can we use `voption`?
```suggestion
  IServiceProvider -> JsonDocument voption -> Task<JsonDocument voption>
```

Reply:
I don't have a strong opinion here, so, yes I can do that. Just out of curiosity: have you ever analyzed an FSharp code for performance and noticed that it's always a good idea to prefer a `voption` over the normal `option`?
I wonder why `voption` is not taught as the norm.

Reply:
Just because it was first I guess 🙂

Reply:
According to the [documentation](https://learn.microsoft.com/en-us/dotnet/fsharp/language-reference/value-options):
_Not all performance-sensitive scenarios are "solved" by using structs. You must consider the additional cost of copying when using them instead of reference types. However, large F# programs commonly instantiate many optional types that flow through hot paths, and in such cases, structs can often yield better overall performance over the lifetime of a program._
I'm not sure if this is a hot path. Would you like to reconsider your suggestion or you are sure that it's going to be a clear benefit to use this special kind of `option` here?

Reply:
In my opinion `option` is a big mistake and should only exist in scenarios where struct is not possible 🙂

Reply:
Fine by me. You changed it yourself, so I'm resolving this conversation.